Tutorials/intro-to-octave/octave_introduction.py

```diff
@@ -23,7 +24,8 @@
 opx_port = 80
 octave_ip = "172.0.0.1"
 octave_port = 50
-
+con = "con1"
+octave = "octave1"
```

Comment:
Parametrizing the controller name makes sense, but this is just a name that the user can define the way he/she wants, so why do we need to parametrize it here? (I just tested and I can do octave="superman")
If you want to test a 2nd Octave, then you need to change the port, no?

Reply:
If you have an other octave so you need to change the octave name and port.. Did you try changing only the port and it worked?

Reply:
I don't remember if I tested it or not, but I don't see any reasons why it shouldn't because the name can be anything (it is not linked to the port)

Reply:
I think you're right